components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIManagerConfiguration.java

```diff
@@ -87,6 +87,25 @@
     private ThrottleProperties throttleProperties = new ThrottleProperties();
     private WorkflowProperties workflowProperties = new WorkflowProperties();
     private Map<String, Environment> apiGatewayEnvironments = new LinkedHashMap<String, Environment>();
+    public  static Map<String, Properties> tokenRevocationNotifiers = new LinkedHashMap<String, Properties>();
```

Comment:
since we now have only 2 notifiers, do we still need a map? can't just use 2 variables?

Reply:
It would be easy to pass the revoked token notifiers as a map to the extended TokenRevocationNotifierImpl class, otherwise we have to pass 2 property variables.

Reply:
I think you have to pass those separately. with that, you can remove all "for each"s in your code. And I noticed you're not passing the property map to methods in the interface. i think we should do that.

Reply:
Instead of `APIManagerConfiguration.getTokenRevocationNotifiers()` you can have 2 methods.

Reply:
ahh,okay, will change accordingly

Reply:
resolved